src/sidebars/star/star.html

```diff
@@ -1,2 +1,20 @@
 ﻿<img class="star" ng-src="assets/png/{{starCtrl.favouriteUrl()}}.png" ng-click="starCtrl.click(); $event.stopPropagation();"
      ng-mouseenter="starCtrl.mouseEnter()" ng-mouseleave="starCtrl.mouseLeave()" draggable="false" />
+
+<div class="confirmation-backdrop " ng-if="starCtrl.confirmationShow" ng-click="starCtrl.hideModal()">
+  <div class="confirmation-modal">
+    <div class=modal-content>
+      <div class="header">
+        <span class="title">StockFlux</span>
+        <icon name="close" icon-click="starCtrl.hideModal()"></icon>
+      </div>
+      <div class="message">
+        <p ng-bind="starCtrl.confirmationMessage"></p>
+      </div>
+      <div class="buttons">
+        <button class="btn btn-primary ng-binding" ng-click="starCtrl.deselect(); $event.stopPropagation();">OK</button>
+        <button class="btn btn-default ng-binding" ng-click="starCtrl.hideModal(); $event.stopPropagation();">Cancel</button>
```

Comment:
I thought `btn` and `btn-primary` etc were bootstrap classes, but we're not using bootstrap. Are they doing something here?